### ISSUE TITLE
Failing to build on linux pt2

### ISSUE THREAD

**Issue by lukassup** [rwengine/openrw#743]
After another attempt to compile fixes in #742 I am still getting some more compile errors:

```
[ 25%] Building CXX object rwengine/CMakeFiles/rwengine.dir/src/audio/SoundBufferStreamed.cpp.o
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/audio/SoundBufferStreamed.cpp: In member function ‘virtual void SoundBufferStreamed::play()’:
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/audio/SoundBufferStreamed.cpp:76:23: error: ‘sleep_for’ is not a member of ‘std::this_thread’
   76 |     std::this_thread::sleep_for(kTickFreqMs);
      |                       ^~~~~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/audio/SoundBufferStreamed.cpp: In member function ‘void SoundBufferStreamed::updateBuffers()’:
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/audio/SoundBufferStreamed.cpp:138:27: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  138 |         std::this_thread::sleep_for(kTickFreqMs);
      |                           ^~~~~~~~~
make[2]: *** [rwengine/CMakeFiles/rwengine.dir/build.make:216: rwengine/CMakeFiles/rwengine.dir/src/audio/SoundBufferStreamed.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:248: rwengine/CMakeFiles/rwengine.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

Info

```
% uname -a
Linux marble 5.14.5-arch1-1 #1 SMP PREEMPT Thu, 16 Sep 2021 11:02:40 +0000 x86_64 GNU/Linux

% gcc --version
gcc (GCC) 11.1.0
...

% make --version
GNU Make 4.3
Built for x86_64-pc-linux-gnu

% cmake --version
cmake version 3.21.2
```

**Comment by lukassup** [comment]
Fix provided in #740

**Comment by tsjost** [comment]
Thanks, fix merged!
